### ISSUE TITLE
RAM Monitor Plugin like lxpanel?

### ISSUE THREAD

**Issue by jacobdesforges** [raspberrypi-ui/wf-panel-pi#18]
In lxpanel, the "Resource Monitor" plugin has a configuration option to visually show the percentage of RAM currently in use. On mouseover, the plugin tells the user how many MB of RAM is in use and what percentage of the total is in use. Obviously, this plugin is only available to RPi users who revert to the X11 desktop. 

Is there a plan to replicate this feature in wf-panel-pi, especially now that Wayland appears to be the future of RPi OS?

It would be nice to see all of the system metrics at a glance in the taskbar, and given that CPU %, CPU Temp, and GPU % are already available in wf-panel-pi, a RAM monitor plugin would round out the system metrics of concern to most users.

**Comment by spl237** [comment]
It's not planned at present, but what I am currently working on is modifying the panel so that it is easy to write and add additional third party plugins, so even if we don't do it, it should hopefully be possible for someone else to!

**Comment by Botspot** [comment]
Yes please consider adding this. I used the RAM monitor constantly to check for memory leaks and to check if I can attribute lagginess to having too many browser tabs open.

Most useful of all is when the system freezes, if panel ram monitor showed it was near 100%, then the freeze could be attributed to OOM.

**Comment by L4wnmower** [comment]
More plugins are definitely welcome. Besides a basic keyboard layout indicator that is a must for multi-layout users, an improved clock showing the weather on the tooltip would be a nice addition

**Comment by Botspot** [comment]
> [spl237](https://github.com/spl237) closed this as [completed](https://github.com/raspberrypi-ui/wf-panel-pi/issues?q=is%3Aissue%20state%3Aclosed%20archived%3Afalse%20reason%3Acompleted)

Which branch was this improvement made to?
